das/das_test.go

```diff
@@ -24,7 +24,7 @@ func TestDASStoreRetrieveMultipleInstances(t *testing.T) {
 	ctx := context.Background()
 
 	messageSaved := []byte("hello world")
-	cert, err := das.Store(ctx, messageSaved)
+	cert, err := das.Store(ctx, messageSaved, CALLEE_PICKS_TIMEOUT)
```

Comment:
Let's add testing where the caller picks the timeout, to make sure the timeout is reflected correctly in the resulting certificate.

Reply:
This will be needed anyway, assuming we eliminate the CALLEE_PICKS_TIMEOUT feature.